.github/workflows/bump_release_version.yml

```diff
@@ -0,0 +1,35 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Not all Tag should be.
There should be 'bump-version-to-*'

Usage:
On local you create tag `bump-version-to-10.5` and after you push it  GitHub, action is started. Action should take release version as `10.5`

Reply:
Not really working for me. I did it but it did not triggered.

Reply:
lets convert this action to use human trigger - https://github.com/checkstyle/checkstyle/pull/11993#discussion_r942433823